src/main/java/dev/jbang/cli/BaseScriptCommand.java

```diff
@@ -153,10 +157,19 @@ public static Script prepareScript(String scriptResource, List<String> arguments
 			s.setAdditionalClasspaths(classpaths);
 
 			List<Path> resolvedSourcePaths = new ArrayList<>();
-			for (FileRef collectSource : s.collectSources()) {
```

Comment:
why move away from using FileRef ? where FileRef then resolves the relative location ?

Reply:
The `Util.collectSources` returns `List<String>` instead `List<FileRef>` because I didn't see the need to convert to FileRef only to then `collectSource.getDestination()`

But I may be missing something. I see that `toFileRef` has a split:
```java
String[] split = fileReference.split("=");
```
So probably my change can break some case that is not in the test set.
If you could send me one test to add please.

I'll change Util.collectSources to return `List<FileRef>` as it's done in `Script.collectSources` to avoid any problems, ok?